content/guides/anomalies.md

```diff
@@ -56,10 +56,13 @@ Continue with steps (3) and (4) as you would for any other monitor.
 
 ### 3. Anomaly Detection Algorithms
 
-We currently offer two different anomaly detection algorithms. Both algorithms will follow the seasonal pattern of the series, and
-will not have their predictions be affected by short anomalies.
+We currently offer four different anomaly detection algorithms. All of the algorithms use past history when forming forecasts, however Robust, Agile, and Adaptive detect and use any additional seasonal structure present in the series when forming their predictions.
```

Comment:
I thought we were discontinuing adaptive?

I would axe the sentence starting with "All of ...". It's obvious that the algorithms use the past, and the differences are already described below.